oe_authentication.install

```diff
@@ -38,4 +41,64 @@ function oe_authentication_install() {
 
     \Drupal::messenger()->addMessage('Please be aware that oe_authentication makes changes to the following configuration: CAS settings.');
   }
+
+  _oe_update_eulogin_fields();
+}
+
+/**
+ * Implements hook_update_N().
```

Comment:
I think we don't need this. At least core doesn't do it. Also in drush it will print it, so I think it is better to leave informative comment here only.